rfcs/067-prismic-api-ids/README.md

```diff
@@ -9,12 +9,19 @@
 We’d like to make sure our Prismic API IDs are written with consistent casing and that we’re following defaults from Prismic so that we’re going with the grain as far as possible and not having to manually override auto-generated SliceMachine files.
 
 ## Custom type API IDs
-These should be kebab-cased and plural when the type is ‘reusable’ as opposed to ‘single’ (e.g. the reusable ‘Exhibition highlight tour’ Custom type has the API ID `exhibition-highlight-tours`, whereas the single ‘Global alert’ Custom type has the API ID `global-alert`).
+When we started using Prismic we decided that these should be kebab-cased. Also, if the type was ‘reusable’ it should be plural, if it was 'single' it should be singular. For example, the reusable 'Exhibition highlight tour' type has an API ID of `exhibition-highlight-tours` whereas the single 'Global alert' type has an API ID of `global-alert`.
 
-SliceMachine makes these snake_case by default but allows this to be overridden easily in the GUI - we've done this fairly consistently although a couple of reusable content types have been given singular API IDs – `card` and `collection-venue`. I don't think there's any mileage in updating everything to snake_case since *none* of them are in that format currently. I'm not sure if we should try to update to `cards` and `collection-venues` or just live with those as they are?
+SliceMachine makes these snake_case by default but allows this to be overridden easily in the Graphical User Interface (GUI) and to-date we have made *all* of them kebab-case, so there probably isn't a good case for changing these to snake_case now.
+
+A couple of reusable content types have been given singular API IDs – `card` and `collection-venue` – should we consider changing these two to `cards` and `collection-venues`?
```

Comment:
If the effort level is low, I think it would be nice, so might be a question of weighing time/effort? With the migration script, is this something we can consider doing easily?

Reply:
Creating the new (pluralised) types and migrating the content should be straightforward. I think the riskier part would be finding where the old (singular types) are linked from and updating those too. e.g. where a card is used on a page, we'd need to link it to the new version of the card

Reply:
Migration doesn't allow for overwriting, right? We always have to re-create a new document?

Reply:
Nice, that'd be great if it worked for IDs